Automattic/pocket-casts-android
# Setup configure Gradle plugin

# Description

This PR adds [configure Gradle plugin](https://github.com/Automattic/configure) and sets up an initial `.configure` file that copies an empty `example-secret` file.

This is my first time setting up `.configure` in a new repo, so in order to make sure I get the initial setup right, I am splitting this step from setting up the actual secrets we'll use.

Here is what I did:
* Created a new key and added it to `keys.json` in our `mobile-secrets` repo
* Created an empty file `android/pocket-casts/example-secret` in `mobile-secrets` repo
* Added the Gradle plugin in 53928f6a6cfd62de52dae634b8b2c7d5a614248b and git-ignored the `vendor/` folder since the plugin will be copying to `configure` binary there
* Created a simple `.configure` file in 363ac2ba45ef2aa287014c95101c9d5b464c683d

Note that I'll remove the encrypted `example-secret` once we add the real secrets.

# To test

* Run `./gradlew applyConfiguration`
* Verify that an empty `example-secret` file is copied to the root of the project
* Remove the `example-secret` file

# Checklist
